app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
     package="com.greenfox.tribesoflagopusandroid">
     <uses-permission android:name="android.permission.INTERNET" />
     <application
+        android:name=".MyApp"
```

Comment:
@georgezsiga can we name our Application class something different? eg `TribesApplication`